crates/transport-http/Cargo.toml

```diff
@@ -35,7 +35,7 @@ hyper = { workspace = true, default-features = false, optional = true }
 hyper-util = { workspace = true, features = ["full"], optional = true }
 
 [features]
-default = ["reqwest", "reqwest-default-tls"]
+default = ["reqwest", "reqwest-default-tls", "hyper"]
```

Comment:
can you compare how many additional deps/build time this will now use as default?

Reply:
Adds about ~20s to build time for `transport-http` crate along with 21 deps.